imgui_impl_vitagl.cpp

```diff
@@ -425,6 +444,14 @@ void ImGui_ImplVitaGL_TouchUsage(bool val){
 	touch_usage = val;
 }
 
+void ImGui_ImplVitaGL_FrontTouchpadMode(bool val){
+	front_touchpad_mode = val;
+}
+
+void ImGui_ImplVitaGL_RearTouch(bool val){
+	rear_touch = val;
+}
+
 void ImGui_ImplVitaGL_KeysUsage(bool val){
```

Comment:
I added user functions `ImGui_ImplVitaGL_FrontTouchpadMode` and `ImGui_ImplVitaGL_RearTouch` to control touch behavior.